docs/plugins/plugin-writer/basics.rst

```diff
@@ -61,8 +65,34 @@ of a plugin app, and so on.
 This matching of module names is required for the Pulp Core to be able to auto-discover
 plugin components, particularly for both models and viewsets.
 
-Subclassing Importer, Publisher
--------------------------------
+Take a look at `the structure of our Example plugin <https://github.com/pulp/pulp_example/tree/master/pulp_example/app>`_.
```

Comment:
Overall can we /our Example plugin/pulp_example/ ? And then near the top have 1 sentence which says that we have an example plugin called pulp_example? What do others think about this?

Reply:
Sounds good to me, no objections.